message_ix/tests/test_integration.py

```diff
@@ -49,9 +51,14 @@ def test_run_clone(tmpdir, request):
     )
 
 
-def test_run_remove_solution(test_mp, request):
+# TODO ixmp4-remove_solution() doesn't seem to remove the timeseries values for
```

Comment:
If I read the test correctly, the time-series/IAMC-structured data should not be changed when model solution data is removed. So I don't think this requires an upstream adjustment.